index.html

```diff
@@ -44,69 +45,77 @@ <h2 class="display-2">onSwipe</h2>
         <!-- JOBS SHOW UP HERE! -->
         <div id="jobs-container" class="row"></div>
       </div>
     </div>
-    
-    <!-- Contact Info section (added by Josh) -->
+
+    <!-- Contact Info section (added by Josh) 
     <div class="row">
       <div class="col-12 col-md-6">
         <h2 class="text-center">Contact Info</h2>
         <ul class="list-unstyled">
           <li><strong>Main Location:</strong> 3423 Piedmont Road NE, Atlanta, GA
             30305</li>
-            <li>
-              <strong>Email: </strong>
-              <a
-              href="mailto: joshua.ellis.r@gmail.com">joshua.ellis.r@gmail.com</a>,
-              <a href="mailto: #">Lakeysha's Email Here</a>,
-              <a href="mailto: #">Max's Email Here</a>
-            </li>
-          </ul>
-        </div>
-        <div class="col-12 col-md-6" id='map' style='width: 200px; height: 200px;'></div>
+          <li>
+            <strong>Email: </strong>
+            <a href="mailto: joshua.ellis.r@gmail.com">joshua.ellis.r@gmail.com</a>,
+            <a href="mailto: #">Lakeysha's Email Here</a>,
+            <a href="mailto: #">Max's Email Here</a>
+          </li>
+        </ul>
       </div>
-      
-      </section>
+      <div class="col-12 col-md-6" id='map' style='width: 200px; height: 200px;'></div>
+    </div> --->
+
+  </section>
 
 
   <!-- Mapbox scripts -->
-  <script
-    src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.js'></script>
+  <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.js'></script>
   <script src="./mapbox.js"></script>
 
-    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
-      integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
-      crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
-      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
-      crossorigin="anonymous"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
-      integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
-      crossorigin="anonymous"></script>
+  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+    crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+    crossorigin="anonymous"></script>
 
-      <script src="./main.js"></script>
+  <script src="./main.js"></script>
 
 </body>
+<footer class="footer">
```

Comment:
I think the footer needs to go within the body element? Just before all the scripts. Does it work like this?

Reply:
It does go inside the body element and I will make sure to put it inside the body.